docs/guides/knowledge_distillation.md

```diff
@@ -52,7 +52,7 @@ git clone https://github.com/AI-Hypercomputer/maxtext.git
 python3 -m venv ~/venv-maxtext
 source ~/venv-maxtext/bin/activate
 cd maxtext
-pip install -r requirements.txt
+uv pip install -r requirements.txt
```

Comment:
Customers will still be able to run without `uv`, right? It's just that you are saying `uv` should be recommended now?